### PR TITLE
feature/lazy-loading

### PR DESCRIPTION
**Pull request by kuzkov**

### CODE REVIEW

File: src/app/pages/login/sign-in/sign-in.component.html [hunk 1]
```diff
@@ -43,9 +43,9 @@ <h2 class="sign-in__title">Sign In</h2>
     <p class="sign-in__sign-in-continue">or continue with</p>
 
     <div class="sign-in__icons-container">
-      <a href="#"><img src="../../../assets/images/Facebook.svg" /></a>
-      <a href="#"><img src="../../../assets/images/Twitter.svg" /></a>
-      <a href="#"><img src="../../../assets/images/Google.svg" /></a>
+      <a href="#"><img src="../../../../assets/images/Facebook.svg" /></a>
```
Comment (marinbonibon) [inline src/app/pages/login/sign-in/sign-in.component.html:46]: and here too many dots)

File: src/app/shared/services/find-movie.service.ts [hunk 2]
```diff
@@ -13,9 +13,9 @@ export class FindMovieService {
   constructor() {}
 
   mediaItems;
-  findMovies: Object[];
-  status: Number;
-  inputValue: String;
+  findMovies: any[];
```
Comment (marinbonibon) [inline src/app/shared/services/find-movie.service.ts:16]: why array of something is called findMovies? It's suitable name for method that is looking for movies

File: src/app/pages/all-search-results/watch-list-button/watch-list-button.component.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@import '../../../styles/vars';
+@import '../../../../styles/vars';
```
[the remark this answers is not in the document]
Comment (kuzkov) [reply src/app/pages/all-search-results/watch-list-button/watch-list-button.component.scss:1]: This one will be deleted

[the remark this answers is not in the document]
Comment (kuzkov) [reply src/app/pages/all-search-results/watch-list-button/watch-list-button.component.scss:1]: i mean this component

File: src/app/domain/navbar/profile-dropdown-menu/profile-dropdown-menu.component.scss [hunk 4]
```diff
@@ -50,14 +50,14 @@
     display: block;
     width: 24px;
     height: 24px;
-    background: url('~assets/images/profile-icon.svg');
+    background: url('../../../../assets/images/profile-icon.svg');
```
Comment (marinbonibon) [inline src/app/domain/navbar/profile-dropdown-menu/profile-dropdown-menu.component.scss:53]: what about these dots?

File: src/app/domain/navbar/profile-dropdown-menu/profile-dropdown-menu.component.scss [hunk 5]
```diff
@@ -50,14 +50,14 @@
...
     margin-right: 10px;
   }
   &__settings-icon {
     display: block;
     width: 24px;
     height: 24px;
-    background: url('~assets/images/settings-icon.svg');
+    background: url('../../../../assets/images/settings-icon.svg');
```
Comment (marinbonibon) [inline src/app/domain/navbar/profile-dropdown-menu/profile-dropdown-menu.component.scss:60]: and these?)

File: src/app/shared/services/find-movie.service.ts [hunk 6]
```diff
@@ -36,23 +37,23 @@ export class FindMovieService {
     }
   }
 
-  getMovies(): Object[] {
+  getMovies(): any[] {
```
Comment (marinbonibon) [inline src/app/shared/services/find-movie.service.ts:40]: why none of methods and variable has accessors? Is it private or public?

**Review by marinbonibon — APPROVED** [review]

**Pull request merged**